Helper/PathHelper.php

```diff
@@ -12,10 +12,14 @@ class PathHelper
     /**
      * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(ContainerInterface $container, $symfony_version)
```

Comment:
What is the reasoning for injecting the Symfony version? I would just use the constant below.